src/sim65/6502.c

```diff
@@ -215,7 +215,9 @@ int PrintCycles;
     ZPAddr = MemReadByte (Regs.PC+1);                           \
     Addr = MemReadZPWord (ZPAddr);                              \
     if (PAGE_CROSS (Addr, Regs.YR))                             \
+    {                                                           \
```

Comment:
This
https://cloud.githubusercontent.com/assets/2664009/24075733/2453022a-0c21-11e7-99e4-a746bb66987d.png
...was reasoned by C&P, but as you may noticed I changed in the mean time all occurrences... (hopefully)